src/Commands/RebuildOaiEntries.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Drupal\islandora_drush_utils\Commands;
+
+use Drupal\Core\Queue\QueueFactory;
+use Drush\Commands\DrushCommands;
+
+/**
+ * RebuildOaiEntries commands.
+ *
+ * These commands rebuild the OAI entries and consume them.
+ */
+class RebuildOaiEntries extends DrushCommands {
+
+  /**
+   * The queue service.
+   *
+   * @var \Drupal\Core\Queue\QueueFactory
+   */
+  protected $queue;
+
+  /**
+   * Set variables to used across commands.
+   *
+   * @param \Drupal\Core\Queue\QueueFactory $queue
+   *   The queue service.
+   */
+  public function __construct(QueueFactory $queue) {
+    $this->queue = $queue;
+  }
+
+  /**
+   * Rebuild OAI entries.
+   *
+   * @command islandora_drush_utils:rebuild-oai
+   * @aliases idr:roai
+   */
+  public function rebuild() {
+    rest_oai_pmh_cache_views();
+
+    $operations = [];
+    $queue = $this->queue->get('rest_oai_pmh_views_cache_cron');
+    while ($item = $queue->claimItem()) {
+      $operations[] = [
+        'rest_oai_pmh_process_queue',
+        [$item],
+      ];
+    }
```

Comment:
Instead of generating multiple discrete operations, having a singular batch operation be re-entrant is preferred, to:
- to avoid memory issues with potentially huge batches being built/serialized
- avoid having the items "claimed" for the default lease time of 30 seconds, but with a sufficiently large amount of items being processed, potentially attempting to process after the lease has elapsed.

So yeah, should roll another method to use as the batch operation, which essentially just:
- attempts to claim an item from the queue
  - if we failed to find an item, return/set the batch to be "finished" (with `$context['finished']` either unset entirely or `1`)
  - if we find an item, process it, and indicate that the batch is not "finished" (with `$context['finished']` between `0` and `1`).

If we wanted to split to multiple commands: One to perform the enqueueing, and one to work as the queue processor, we would be able to employ multiprocessing mechanisms (say, using GNU parallel) to be able to have multiple workers processing the queue, but is admittedly stepping somewhat outside of the described scope of this ticket.